### PR TITLE
Change HorizontalPodAutoscaler walkthrough to use autoscaling/v2 API group

### PR DESCRIPTION
**Pull request by MeenuyD**
Horizontal Pod Autoscaler API version autoscaling/v2 , and it provides more features than autoscaling/v1.
#43685

### CODE REVIEW

**Comment by Shubham82** [comment]
Thanks @MeenuyD 
The preview looks good to me:
https://deploy-preview-43688--kubernetes-io-main-staging.netlify.app/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/

/lgtm

**Review by aj11anuj — APPROVED** [review]
LGTM too, 👍

**Comment by sftim** [comment]
I think this will be clearer:
/retitle Change HorizontalPodAutoscaler walkthrough to use autoscaling/v2 API group

**Comment by sftim** [comment]
> Horizontal Pod Autoscaler API version autoscaling/v2beta2 is in beta, and it provides more features than autoscaling/v1

v2beta2 is out of beta. Would you update the PR description @MeenuyD?

**Comment by MeenuyD** [comment]
Thank you for correcting me and sorry for my confusion

**Comment by garritfra** [comment]
> Thank you for correcting me and sorry for my confusion

@MeenuyD did you get a chance to update the PR description yet? 🙂

**Comment by Shubham82** [comment]
> > Thank you for correcting me and sorry for my confusion
> 
> @MeenuyD did you get a chance to update the PR description yet? 🙂

@garritfra, The PR description was updated by @MeenuyD.

**Comment by Shubham82** [comment]
@sftim could you approve this PR so that it will merge?

Thanks!

**Comment by sftim** [comment]
> @sftim could you approve this PR so that it will merge?

The PR also needs LGTM from a reviewer (so, one LGTM, and another person adds their approval). That's because this change will go live as soon as we merge it.

I'll take a look and see if I'm happy to add that LGTM.
However, @Shubham82, please limit the amount of times you ask for reviews from specific individuals. You should aim to do that only when other reviewers might not have the appropriate context, or when there are other good reasons to focus on a subset of the available reviewers.
Otherwise, the request might be seen as a sign that other contributors should step back, and that can slow the review process.

**Review by sftim — COMMENTED** [review]
LGTM for docs

How did you test this change @MeenuyD?

**Comment by MeenuyD** [comment]
Apply the updated HorizontalPodAutoscaler manifest to the Kubernetes cluster.

**Comment by Shubham82** [comment]
> I'll take a look and see if I'm happy to add that LGTM. However, @Shubham82, please limit the amount of times you ask for reviews from specific individuals. You should aim to do that only when other reviewers might not have the appropriate context, or when there are other good reasons to focus on a subset of the available reviewers. Otherwise, the request might be seen as a sign that other contributors should step back, and that can slow the review process.

Hi @sftim, I didn't mean that, i only want if everything is fine so could we merge this PR But I will take care of this thing next time.

Thanks for this information.

**Comment by divya-mohan0209** [comment]
@kubernetes/sig-autoscaling-pr-reviews : PTAL & signoff from a tech standpoint.

**Comment by divya-mohan0209** [comment]
I have zero idea! I literally put a do not merge label on there and it still merged 🤷

**Comment by BenTheElder** [comment]
> How come this merged?

[This is from my past recollection working on prow, it's likely this is still true but possible something has changed github side or prow side]

Because the hold was put in place after the PR became eligible for merge and github can lag on indexing the information + the merge robot has an update loop to scan for mergeable PRs. (The API used to search for mergeable PRs relies on a search index which can lag slightly behind)

Trying to hold at the last minute creates a race that is occasionally lost, I recommend holding in the same or earlier comment as approving to avoid this.

**Comment by divya-mohan0209** [comment]
Alright, thanks for confirming @BenTheElder ! I'll keep this in mind :)

**Pull request merged**
